q2_types/sample_data/tests/test_transformer.py

```diff
@@ -26,7 +26,8 @@ def test_pd_series_to_alpha_diversity_format(self):
                         name='shannon', index=exp_index)
 
         obs = transformer(exp)
-        obs = pd.Series.from_csv(str(obs), sep='\t', header=0)
+        obs = pd.read_csv(str(obs), sep='\t', parse_dates=True, header=0,
+                          index_col=0, squeeze=True)
```

Comment:
Why are you setting `squeeze=True`?

Reply:
This [link](https://pandas.pydata.org/pandas-docs/version/0.21/generated/pandas.Series.from_csv.html#pandas.Series.from_csv) talked about including parameter values that would be needed to return a series similar to what `from_csv` accomplished. 

I mistook these as necessary keyword argument values, rather than suggested values in place of defaults or other values. I will remove them from the function call.